README.md

```diff
@@ -331,6 +331,34 @@ You can specify the output files location with the `-o` option:
 tondev sol compile Contract.sol -o path/to/output/file
 ```
 
+#### Ast-json
+
+This command parses a ton-solidity file and creates an abstract syntax tree (AST) to the output directory.
+
+```shell
+tondev sol ast-json Contract.sol
+```
+
+Contract.ast.json file will be saved into the current directory.
+
+You can specify the output file location with the `-o` option:
+
+```shell
+tondev sol ast-json Contract.sol -o path/to/output/file
+```
+
+#### Ast-compact-json
+
+The same as previous but save AST in compact JSON form.
+
+```shell
+tondev sol ast-compact-json Contract.sol
```

Comment:
I think it is better to name command as `ast` and add option `-f, --format <json | compact-json>`